rules/integrations/aws/privilege_escalation_aws_suspicious_saml_activity.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/09/22"
+maturity = "production"
+updated_date = "2021/09/22"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when suspicious SAML activity has occurred in AWS. An adversary could gain backdoor access via SAML manipulation.
+"""
+false_positives = [
+    """
+    SAML Provider could be updated by a system administrator. Verify whether the user identity, user agent, and/or 
+    hostname should be making changes in your environment. SAML Provider being updated from unfamiliar users should 
+    be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-25m"
+index = ["filebeat-*", "logs-aws*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS Suspicious SAML Activity"
```

Comment:
If we don't differentiate between legit activity, we should remove the "suspicious" here